internal/jimm/jimm.go

```diff
@@ -140,6 +141,12 @@ type GroupManager interface {
 	CountGroups(ctx context.Context, user *openfga.User) (int, error)
 }
 
+type IdentityManager interface {
```

Comment:
godoc.. but if we exported the type in the identity package, we wouldn't need this interface..